python/albert/__init__.pyi

```diff
@@ -0,0 +1,345 @@
+                   object representing the current query execution. This
+                   function should return a list of Item objects.  See the Item
+                   class section below.
+initialize()       Optional. This function is called when the extension is
+                   loaded. Although you could technically run your
+                   initialization code in global scope, it is recommended to
+                   initialize your extension in this function. If your extension
+                   fails to initialize you can raise exceptions here, which are
+                   displayed to the user.
+finalize()         Optional. This function is called when the extension is
+                   unloaded.
+"""
+from enum import Enum
+from typing import Any
+from typing import Callable
+from typing import List
+from typing import Optional
+from typing import Union
+
+
+def debug(arg: Any) -> None:
+    """
+    Log a message to stdout. Note that debug is effectively a NOP in release
+    builds. Puts the passed object into str() for convenience. The messages are
+    logged using the QLoggingCategory of the python extension and therefore are
+    subject to filter rules.
+    """
+
+
+def info(arg: Any) -> None:
+    """
+    Log a message to stdout at the "info" log level. Puts the passed object into
+    str() for convenience. The messages are logged using the QLoggingCategory of
+    the python extension and therefore are subject to filter rules.
+    """
+
+
+def warning(arg: Any) -> None:
+    """
+    Log a message to stdout at the "info" log level. Puts the passed object into
+    str() for convenience. The messages are logged using the QLoggingCategory of
+    the python extension and therefore are subject to filter rules.
+    """
+
+
+def critical(arg: Any) -> None:
+    """
+    Log a message to stdout at the "info" log level. Puts the passed object into
+    str() for convenience. The messages are logged using the QLoggingCategory of
+    the python extension and therefore are subject to filter rules.
+    """
+
+
+def iconLookup(iconName: Union[str, List[str]]) -> Union[str, List[str]]:
+    """
+    Perform xdg icon lookup and return a path. Empty if nothing found.
+    This function has two forms. Called with a single string argument, it
+    returns a single (possibly empty) string. Called with a list of string
+    arguments, it returns a list of (possibly empty) string results.
+    """
+
+
+def cacheLocation() -> str:
+    """
+    Returns the writable cache location of the app. (E.g. $HOME/.cache/albert/
+    on Linux)
+    """
+
+
+def configLocation() -> str:
+    """
+    Returns the writable config location of the app. (E.g. $HOME/.cache/albert/
+    on Linux)
+    """
+
+
+def dataLocation() -> str:
+    """
+    Returns the writable data location of the app. (E.g. $HOME/.cache/albert/
+    on Linux)
+    """
+
+
+class Query(object):
```

Comment:
Useless object inheritence. All classes in Python3 inherited from it by default.